### PR TITLE
OKTA-702581 Updates to OIN Wizard backward compatibility section 

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Updates to OIN Wizard backward compatibility section for SAML and OIDC
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-702581](https://oktainc.atlassian.net/browse/OKTA-702581)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/openidconnect/backward-compatible-eg.md](https://github.com/okta/okta-developer-docs/blob/889d7be0451faf01ad64dc3a20aaac17dc73b7ed/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-oin-app%2Fmain%2Fopenidconnect%2Fbackward-compatible-eg.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/13a8181b-0e8a-459d-a45b-65d8f7aa4109) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/saml2/backward-compatible-eg.md](https://github.com/okta/okta-developer-docs/blob/889d7be0451faf01ad64dc3a20aaac17dc73b7ed/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-oin-app%2Fmain%2Fsaml2%2Fbackward-compatible-eg.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/13ee4609-791b-4f2a-abf9-7effcf3809a2) | 100  | 100  | 100  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.21d0a119-225d-4d43-8738-5146c3e2b6d5?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/saml2/backward-compatible-eg.md [hunk 1]
```diff
@@ -3,7 +3,9 @@
...
    ```bash
-   ${empty org.companyId ? 'https://login.myapp.io' : 'https://login.myapp.io?connection=' + org.companyId}
+   ${empty org.companyId ? 'login.myapp.io' : 'login.myapp.io?connection=' += org.companyId}
    ```
 
-   This expression handles both scenarios where `companyId` is populated or empty.
\ No newline at end of file
+   This expression handles both scenarios where `companyId` is populated or empty.
+
+   > **Note:** Only HTTPS URLs are allowed for ACS URLs. You don't need to add `https://` to the URL string.
```
Comment (jackward-okta) [inline packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/saml2/backward-compatible-eg.md:11]: Wow, didn't know this! Did you get this from experience or elsewhere in our docs? I've typically been including `https://` in my ACS URLs and I had a vague memory that we required it but don't have an environment up right now to check.

File: packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/saml2/backward-compatible-eg.md [hunk 2]
```diff
@@ -3,7 +3,9 @@
    To handle empty `companyId` values, you can define the ACS URL as:
 
    ```bash
-   ${empty org.companyId ? 'https://login.myapp.io' : 'https://login.myapp.io?connection=' + org.companyId}
+   ${empty org.companyId ? 'login.myapp.io' : 'login.myapp.io?connection=' += org.companyId}
```
Comment (ykrestyanov-okta) [inline packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/saml2/backward-compatible-eg.md:6]: https:// is missed, the expression should be 
`https://${empty org.companyId ? 'login.myapp.io' : 'login.myapp.io?connection=' += org.companyId}`

Comment (ykrestyanov-okta) [reply packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/saml2/backward-compatible-eg.md:6]: @vanngo-okta

File: packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/saml2/backward-compatible-eg.md (hunk 1 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/saml2/backward-compatible-eg.md:11]: Sorry @jackward-okta , my bad: I misunderstood Abbas's comment about this field. I'll remove this note.

File: packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/saml2/backward-compatible-eg.md (hunk 2 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/saml2/backward-compatible-eg.md:6]: Thanks, I'll correct this in another commit.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/openidconnect/backward-compatible-eg.md](https://github.com/okta/okta-developer-docs/blob/07d973f456a2c08db29516603643a610cd692261/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-oin-app%2Fmain%2Fopenidconnect%2Fbackward-compatible-eg.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/7bfa6f5b-5627-4cdf-93ee-66bbfb1979e7) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/saml2/backward-compatible-eg.md](https://github.com/okta/okta-developer-docs/blob/07d973f456a2c08db29516603643a610cd692261/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-oin-app%2Fmain%2Fsaml2%2Fbackward-compatible-eg.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/f3e916b4-104d-48bc-80bb-9ea6f4bf4c5f) | 100  | 100  | 100  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.af748d12-cfc1-4336-a75e-dc9c83295957?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by ykrestyanov-okta — APPROVED** [review]
@vanngo-okta lgtm

File: packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/saml2/backward-compatible-eg.md [hunk 3]
```diff
@@ -3,7 +3,7 @@
    To handle empty `companyId` values, you can define the ACS URL as:
 
    ```bash
-   ${empty org.companyId ? 'https://login.myapp.io' : 'https://login.myapp.io?connection=' + org.companyId}
+   https://${empty org.companyId ? 'login.myapp.io' : 'login.myapp.io?connection=' += org.companyId}
    ```
 
-   This expression handles both scenarios where `companyId` is populated or empty.
\ No newline at end of file
+   This expression handles both scenarios where `companyId` is populated or empty.
```
Comment (jackward-okta) [inline packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/saml2/backward-compatible-eg.md:9]: Huh, I wonder why there's a diff here. End-of-file newline, maybe?

**Review by jackward-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/saml2/backward-compatible-eg.md (hunk 3 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/saml2/backward-compatible-eg.md:9]: Maybe? Let me review the file again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/openidconnect/backward-compatible-eg.md](https://github.com/okta/okta-developer-docs/blob/8bc67cf96a65cd48151316c3b682aedd4998b6bd/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-oin-app%2Fmain%2Fopenidconnect%2Fbackward-compatible-eg.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/495bce86-7939-4eaf-bd6c-cdaba8038fb2) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/saml2/backward-compatible-eg.md](https://github.com/okta/okta-developer-docs/blob/8bc67cf96a65cd48151316c3b682aedd4998b6bd/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-oin-app%2Fmain%2Fsaml2%2Fbackward-compatible-eg.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/3d7f027a-1697-4794-b1f8-43acafc08559) | 100  | 100  | 100  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.0e2cc8bc-12ba-45f5-9d3d-787df90686ae?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/saml2/backward-compatible-eg.md (hunk 3 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/saml2/backward-compatible-eg.md]: Looks like that removed the highlighted change. Thanks for the tip @jackward-okta !

**Review by grahamsmith-okta — APPROVED** [review]

**Pull request merged**